browser/index.html

```diff
@@ -211,6 +211,12 @@
                                                     <li v-if="'vultr' in build.meta.images">
                                                         Vultr: <span v-html="getImageUrl(build, 'vultr')"></span>
                                                     </li>
+                                                    <li v-if="'extensions-container' in build.meta.images">
+                                                        Extensions Container: <span v-html="getImageUrl(build, 'extensions-container')"></span>
+                                                    </li>
+                                                    <li v-if="'legacy-oscontainer' in build.meta.images">
+                                                        Legacy OSContainer: <span v-html="getImageUrl(build, 'legacy-oscontainer')"></span>
+                                                    </li>
```

Comment:
Can you add them in alphabetical order?

Reply:
I started to do that but it got complicated. It looks like we tried to group the Live artifacts together (maybe we can drop the older `iso`, `kernel`, `initramfs` entries now? If we do that then it would get easier.) WDYT?

Reply:
I don't think we support any release with the old artifacts now, so SGTM. But I wouldn't "force group" those two containers together. They're related in that they're both containers, but otherwise independent.

Reply:
fixed

Reply:
I see why I was confused before. The sorting was based on the description name, not the actual name of the key in the json file.

Either way. Fixed now :)